### PR TITLE
Notify dialogflow adapter of team creation.

### PR DESCRIPTION
**Pull request by henleo**

### CODE REVIEW

File: .env.testing [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ES_URL=http://localhost:19200
 NEO4J_URL=bolt://:@localhost:17687
+DIALOGFLOW_ADAPTER=http://www.localhostdrolf.de
```
Comment (Askir) [inline .env.testing]: Some url that tells you what is wrong would be nice here imo. i.e. `tobito.should.be.mocked`

File: application/controllers/teams_controller.py [hunk 2]
```diff
@@ -1,10 +1,13 @@
 """
 Handles all logic of the artefacts api
 """
+import os
+import requests
+
+from webargs.flaskparser import use_args
+#from eventlet import spawn_n
```
Comment (Askir) [inline application/controllers/teams_controller.py]: remove ? :D

File: application/controllers/teams_controller.py [hunk 3]
```diff
@@ -44,6 +47,9 @@ def index(self, params):  # pylint: disable=W0613
     def create(self, params):
         """Logic for creating a team"""
         team = Team(name=params["name"]).save()
+        print(team)
```
Comment (Askir) [inline application/controllers/teams_controller.py]: remove? :D

File: application/controllers/teams_controller.py [hunk 4]
```diff
@@ -44,6 +47,9 @@ def index(self, params):  # pylint: disable=W0613
...
+        self.notify_of_team_creation(team)
+        #spawn_n(self.notify_of_team_creation, team)
```
Comment (Askir) [inline application/controllers/teams_controller.py]: remove? :D or maybe just uncomment if you want to make this threaded now, see other comments

File: specs/requests/teams_spec.py [hunk 5]
```diff
@@ -49,6 +51,24 @@
             with it('saves the team'):
                 expect(Team.nodes.get_or_none(id_=self.response.json["id"], name="My Team")).to_not(be(None))
 
+        with description('notify dialogflow adapter'):
+            with before.each:
+                current_app.config['DIALOGFLOW_NOTIFY'] = True
+                self.DIALOGFLOW_URL = os.environ.get('DIALOGFLOW_ADAPTER') + '/teams'
+                self.rsps = responses.RequestsMock()
+                self.rsps.__enter__()
```
Comment (Askir) [inline specs/requests/teams_spec.py]: This is honestly pretty hacky. If you want to leave it this way at least leave a comment on how it works. Or change it back to: `with responses.RequestsMock() rsps:`

**Review by Askir — COMMENTED** [review]
Aside from a few comments, looks pretty functional. Wont block merge, I think you'll know what to fix. Definitely should implement an observer like pattern for these sort of structures in the future though! If you're a real busy bee you could already create a refactoring issue for that :)

**Comment by Askir** [comment]
Oh also one more thing! please squash the "Fix linting" and similar commits into one.

**Comment by Adrian-St** [comment]
Changes have been merged in another branch

**Pull request closed without merging**